Paws4Me/Paws4Me/Display All Adoptable Pets/View Models/PetAllViewModel.swift

```diff
@@ -18,6 +18,7 @@ class AllPetDataViewModel {
     // MARK: - Vars/Lets
     private var petRepository: PetDataRepository?
     private weak var delegate: PetViewModelDelegate?
+    private lazy var singlePetViewModel = SinglePetViewModel()
```

Comment:
Please have a look at your feature structure `ViewModels` should not reference other `ViewModels`

Reply:
Will do so 👍🏻